src/config.ts

```diff
@@ -13,7 +13,8 @@ export const constants = {
   scmCommandApiPath: '/api/command',
   scmDomain: '.scm.azurewebsites.net',
   scmVfsPath: '/api/vfs/site/wwwroot/',
-  scmZipApiPath: '/api/zip/site/wwwroot/'
+  scmZipApiPath: '/api/zip/site/wwwroot/',
```

Comment:
Do we still need to have the old endpoint const?

Reply:
Good catch, we can remove it